whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/ElasticFulltextSearch.java

```diff
@@ -111,51 +114,70 @@ protected SearchResponse doSearch() throws IOException {
                 final org.elasticsearch.action.search.SearchResponse fulltextResponse = elasticIndexService.getClient().search(fulltextRequest, RequestOptions.DEFAULT);
                 final SearchHit[] hits = fulltextResponse.getHits().getHits();
 
-                LOGGER.debug("ElasticSearch {} hits for the query: {}", hits.length, searchRequest.getQuery());
-                final List<RpslObject> results = Lists.newArrayList();
-                int resultSize = Math.min(maxResultSize,Long.valueOf(fulltextResponse.getHits().getTotalHits().value).intValue());
+                LOGGER.info("ElasticSearch {} hits for the query: {}", hits.length, searchRequest.getQuery());
```

Comment:
Make this DEBUG or we'll be seeing it constantly in the console log

Reply:
oh yes sorry, forgot to revert the change!